server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMessagesMethodTest.java

```diff
@@ -1470,7 +1467,7 @@ public void setMessagesShouldMarkAsDraftWhenIsDraftPassed() {
     }
 
     @Test
-    public void setMessagesShouldRejectCreateInDraftAndOutboxForASingleMessage() {
+    public void setMessagesShouldNotRejectCreateInDraftAndOutboxForASingleMessage() {
```

Comment:
how is it possible ?

Reply:
You're right. I should still ensure that when we create a message in Outbox, it should be only in Outbox.